tests/test-timing.js

```diff
@@ -31,8 +31,19 @@ tape('non-redirected request is timed', function(t) {
     t.equal(err, null)
     t.equal(typeof res.elapsedTime, 'number')
     t.equal(typeof res.responseStartTime, 'number')
+    t.equal(typeof res.timings, 'object')
     t.equal((res.elapsedTime > 0), true)
     t.equal((res.responseStartTime > r.startTime), true)
+    t.equal((res.timings.start > 0), true)
```

Comment:
Yeah, that's why I hadn't used `.deepEqual` initially.  I've added a check to get all property names and compare.